packages/fast-jss-manager-react/src/jss-manager.tsx

```diff
@@ -38,6 +38,16 @@ export type ManagedJSSProps<T, S, C> = Pick<
 > &
     JSSManagedComponentProps<S, C>;
 
+export function mergeClassNames(a: string | void, b: string | void): string {
+    if (typeof a === "string" && typeof b === "string") {
+        return a.concat(" ", b);
+    } else if (typeof a === "string") {
+        return a;
+    } else if (typeof b === "string") {
+        return b;
+    }
+}
```

Comment:
if this function always has to return a string, should it, outside of the if...else blocks `return "";`? Otherwise the return value is not string its potentially `string | void`